### ISSUE TITLE
How to suppress stdout from 'air'?

### ISSUE THREAD

**Issue by jsfag** [air-verse/air#201]
It's freaking annoying:

<img width="525" alt="image" src="https://user-images.githubusercontent.com/3107471/139125918-6ae04297-0581-4d63-9800-b58ed375ac1a.png">

**Comment by silverwind** [comment]
It'd be nice to be able to selectively suppress stuff like the `watching <something>` lines. On big projects, it spams the terminal with hundrets of useless lines which is rather annoying.

**Comment by ThomasJanUta** [comment]
I just started using air. Installing, configuring, running air is great except that you get blasted with noise:
```

  __    _   ___
 / /\  | | | |_)
/_/--\ |_| |_| \_ , built with Go

watching .
watching __pycache__
watching __pycache__
!exclude log
!exclude log
!exclude pkg
watching py
watching py
watching py\__pycache__
watching py\__pycache__
watching py\__pycache__
watching py\secret
watching py\secret
watching py\secret
watching py\util
watching py\util
watching py\util
watching py\util\__pycache__
watching py\util\__pycache__
watching py\util\__pycache__
watching py\util\__pycache__
!exclude res
!exclude res
watching src
watching src\config
watching src\secret
watching src\exec
watching src\http
watching src\util
building...
running...
```
This is all useless including the duplicated paths. I know that I am using AIR, I know how the project looks like.

Adding some kind of silent mode in the configuration would work wonders:
```
[log]
  time = false
  silent = true  # enable silent mode
```
@xiantang @cosmtrek Could you comment where in the code you would inject an "escape" or flag for silencing the stoud?

**Comment by silverwind** [comment]
Maybe log levels could be introduced. I'd classicy the above "watching" stuff as "debug", for example.

**Comment by intel352** [comment]
This is still needed, despite #367.

For example, if using Hashicorp's go-plugin, which relies on stdout, go-plugin functionality is broken because of air controlling output to stdout.

Log levels would be great, but to allow compatibility with repos like go-plugin, a config setting to control where air sends it's output would likely solve the issue, as logging to file-only would avoid the stdout conflict.

**Comment by brettinternet** [comment]
#367 suppresses air's watcher, build, and runner logs but the version info remains. Is there a way to suppress the version information logging?

```
  __    _   ___
 / /\  | | | |_)
/_/--\ |_| |_| \_ v1.52.3, built with Go go1.22.4
```

**Comment by jesses-code-adventures** [comment]
@brettinternet i have put in #641 that allows a separate flag "silent" that suppresses the version info and the notifications about files being watched/refreshing, so the only output you get is from your app.

**Comment by xiantang** [comment]
should be fixed #641
